pc/src/menu.rs

```diff
@@ -29,83 +42,177 @@ const BUTTON_FONT_SIZE: u16 = 16;
 			))
 			.background_margin(RectOffset::new(37.0, 37.0, 5.0, 5.0))
 			.margin(RectOffset::new(10.0, 10.0, 0.0, 0.0))
-			.background_hovered(Image::from_file_with_format(
-				include_bytes!("../assets/img/Menu/button_hovered_background.png"),
-				None
-			))
-			.background_clicked(Image::from_file_with_format(
-				include_bytes!("../assets/img/Menu/button_clicked_background.png"),
+			.text_color(Color::from_rgba(180, 180, 100, 255))
+			.font_size(BUTTON_FONT_SIZE)
+			.build();
+		Skin {
+			window_style,
+			button_style,
+			..root_ui().default_skin()
+		}
+	}
+}
+
+fn get_button_focused_skin() -> Skin {
+	{
```

Comment:
I don't think this extra brace that contains the entire function code serves any purpose